index.js

```diff
@@ -42,46 +42,87 @@ module.exports.pitch = function (remainingRequest) {
   // This prevents [chunkhash] values from changing when running webpack
   // builds in different directories.
   // this.loadModule('@angular/core', (a,b,c,d) =>{debugger;});
-  if (this.query.exclude.some(mdl => this._module.rawRequest.match(new RegExp(mdl)))){
-    return;
-  }
   if (this.query.modules && this.query.modules.length
     && this.query.modules
       .every(mdl => !this._module.rawRequest.match(new RegExp(mdl))))  {
     return;
   }
-  const newRequestPath = remainingRequest.replace(
+
+  let newRequestPath = remainingRequest.replace(
     this.resourcePath,
     '.' + path.sep + path.relative(this.context, this.resourcePath)
   );
 
   this.cacheable && this.cacheable();
   if (!this.query || !this.query.namespace || !this.query.modules) throw new Error("query parameter is missing");
   // Determine how to resolve the global object
+
+  let globalVar;
   let request = this._module.rawRequest.split('!');
-  request = request[request.length - 1].replace(/^@/i, '').replace(/\//g, '.');
-  const globalVar = `${this.query.namespace.replace(/^\?/i, '')}.${request}`;
+
+  if (this._module.userRequest.includes('/node_modules/')) {
+    request = request[request.length - 1].replace(/^@/i, '').replace(/\//g, '.');
+    globalVar = `${this.query.namespace.replace(/^\?/i, '')}.${request}`;
+  } else { //Use modules from parent app
+    request = request[request.length - 1].replace(/^./i, '').replace(/\//g, '.');
+    globalVar = `${this.query.namespace}${request}`;
```

Comment:
should this be: 
`
const globalVar = `${this.query.namespace.replace(/^\?/i, '')}.${request}`;
`